Prolangle/Program.cs

```diff
@@ -41,8 +43,8 @@
 	switch (environment.Environment)
 	{
 		case "Development":
-			currentGameTimestamp = DateTime.Now;
-			nextGameTimestamp = DateTime.Now.AddHours(1);
+			currentGameTimestamp = DateTime.Today.AddHours(DateTime.Now.Hour);
```

Comment:
hm. why this change? I think the idea was that development makes it easier to test under different seeds